sentry-core/build.gradle.kts

```diff
@@ -65,6 +66,18 @@ tasks {
     }
 }
 
+buildConfig {
+    useJavaOutput()
+    packageName("io.sentry.core")
+    buildConfigField("String", "SDK_NAME", "\"${project.name}\"")
```

Comment:
project name will be what here exactly? `sentry-core?
I'm a bit lost but SDK name is `sentry.java`, `sentry.java.android`.. etc. Sentry convention

Reply:
It would be `sentry-core`. Ok, I'll change it to `sentry.java`.

Reply:
maybe the key of this field should be `SENTRY_CLIENT_NAME` so it matches `options.sentryClientName`

Reply:
Actually I missed here setting `options.sentryClientName` - at this moment we are just setting `SdkVersion`. I will align it with Android implementation.

Reply:
ah wait wait, I've mistaken the topic maybe too.
sentryClientName != the sdkVersion.packages.

Reply:
I think you're right in the end. I'll add this line to `SentryOptions`.

```
setSentryClientName(BuildConfig.SENTRY_CLIENT_NAME + "/" + BuildConfig.VERSION_NAME);
```

Reply:
This client name goes into the outgoing HTTP header, right? That to be honest doesn't really matter. We don't use for anything except debuggin (and possibly dropping all events on a load balancer based on this value) but the `sdk name` is important (i.e: sentry.java, sentry.java.android, sentry.java.spring, etc). And packages, are the actual package in the format pkgmanager/pkgname, like maven:io.sentry, maven:io.sentry.android, nuget:Sentry.Serilog